### DISCUSSION TITLE
Private Wiki.js

### DISCUSSION THREAD

**Discussion by maukunze** [requarks/wiki#6147] [Help / Questions] [answered]
i installed wiki js on a ubuntu 22.04 machine and want to make it private.
As stated in the documentation i have to just toggle the Page Rule of the Guest Group to deny or simply delete all global permissions.
As for now both things dont work and when i open up the page without logging in i can see everything in wiki js.
Is there something im missing ?

**Comment by NGPixel** [comment]
- Try restarting Wiki.js
- Try to browse in incognito mode in a different browser to test.

**Comment by maukunze** [comment, marked as answer]
Hey @NGPixel 
yes i tried that but it didnt work.
But after waiting the night it somehow works now. 
Thanks for the input.

**Comment by theAkito** [comment]
Wow! Now, I get it. Was searching the documentation for any keyword related to having a private instance. Didn't find anything. Now, I realise, it's basically a side-effect of the already present fine-grained access control. Great!

It'd be very helpful, if this use-case would be pointed out in the documentation. 🙂

It's a fairly common use-case & searching for guest permissions is probably not the first thing one would search for.
The keywords searched for are usually "login only", "private" or "no landing page", as far as I know.
